langfuse/src/langfuse.ts

```diff
@@ -22,12 +23,22 @@ export class Langfuse extends LangfuseCore {
   private _storageCache: any;
   private _storageKey: string;
 
-  constructor(params: { publicKey: string; secretKey: string } & LangfuseOptions) {
-    super(params);
-    const { publicKey, secretKey, ...options } = params;
+  constructor(params?: { publicKey?: string; secretKey?: string } & LangfuseOptions) {
+    const { publicKey, secretKey, ...options } = utils.configLangfuseSDK(params);
+    if (!secretKey) {
+      throw new Error("secretKey is required");
```

Comment:
I remember that we want to change the default here to logging an error but not throwing an exception if keys are not set. configLangfuseSDK already logs errors